catalog-info.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
       title: "Frontend app library authoring"
       icon: "Web"
   annotations:
-    openedx.org/arch-interest-groups: ""
+    openedx.org/arch-interest-groups: "navinkarkera"
 spec:
   owner: group:teaching-and-learning
```

Comment:
@navinkarkera you should update this item.  This sample file has a bunch of helpful comments: https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.html#references

Reply:
@feanil Updated.